jpw1991/chebs-necromancy
# 1.7.2: allow minions to roam when holding shift+T; fix bug in ZDO comparison

# Pre-compiled

You can get 1.7.3 here to test with https://github.com/jpw1991/chebs-necromancy/releases/tag/1.7.3

# Description

- RoamRange exposed to config
- Using Shift+T on following minions permits them to roam
- Minions remember if they were following a player and continue following when the player logs back in
- 1.7.4: refueler pylons now also refuel fireplaces eg. torches, bonfires, etc. No need to test this one if you don't want to. Already did lots
- 1.7.5: Neckro messages hideable; Neckro messages improved

## Testing

- Ensure that Wait works like before (also on logout/login)
- Ensure that Follow works like before (also on logout/login)
- Ensure that Roam works and that it is remembered on logout/login
- Ensure that minions following the player when player logs off continue following the player after player logs back in
